styles/kotlin/catppuccin.user.css

```diff
@@ -0,0 +1,697 @@
+      background-color: darken(@base, 2%);
+
+      & > div[class*="__checkbox"] > div[class*="_wrapper_"] > label {
+        background-color: darken(@base, 2%)
+      }
+
+      &__icon {
+        @svg: escape('<svg width="96" height="96" viewBox="0 0 96 96" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M40.762 3.592c3.647-4.79 10.829-4.79 14.476 0l1.76 2.313 2.546-1.393c5.27-2.886 11.831.048 13.224 5.914l.672 2.832 2.889-.233c5.984-.483 10.79 4.877 9.686 10.805l-.533 2.862 2.734.967c5.662 2.004 7.881 8.864 4.473 13.828l-1.645 2.397 2.105 2c4.361 4.144 3.61 11.318-1.513 14.46l-2.474 1.518 1.114 2.688c2.306 5.567-1.285 11.814-7.238 12.592l-2.874.375-.072 2.91c-.147 6.029-5.957 10.269-11.71 8.547l-2.778-.831-1.244 2.63c-2.576 5.446-9.6 6.946-14.159 3.023L48 91.9l-2.201 1.895c-4.559 3.923-11.583 2.423-14.16-3.023l-1.243-2.63-2.778.831c-5.753 1.722-11.563-2.518-11.71-8.546l-.072-2.91-2.874-.376c-5.953-.778-9.544-7.025-7.238-12.592l1.114-2.688-2.474-1.517C-.76 57.202-1.51 50.028 2.85 45.885l2.106-2.001-1.646-2.397C-.097 36.523 2.12 29.663 7.784 27.66l2.734-.967-.533-2.862c-1.103-5.928 3.702-11.288 9.686-10.805l2.89.233.672-2.833c1.392-5.865 7.952-8.799 13.223-5.913l2.545 1.393z" fill="@{text}"/><path fill-rule="evenodd" clip-rule="evenodd" d="M44.845 50.523a4.75 4.75 0 0 0 6.31 0L72 32.003V60a4 4 0 0 1-4 4H28a4 4 0 0 1-4-4V32.003zM26.255 32 45.84 49.401a3.25 3.25 0 0 0 4.318 0L69.745 32z" fill="@{mantle}"/></svg>');
+        content: url("data:image/svg+xml,@{svg}")
+      }
+    }
+
+    div.gmnoprint > div {
+
+      & > button {
+        &.gm-control-active {
+          background-color: @base !important;
+
+          &:disabled {
+            background-color: darken(@base, 5%) !important;
+          }
+        }
+      }
+    }
+
+    img[class*="hero_imageDesktop"] {
+      filter: contrast(1.25);
+      mix-blend-mode: screen;
+    }
+
+    .global-layout > div > div {
+      background-color: @base;
+    }
+
+    [class*="_tab_"] {
+      color: @text;
+
+
+      &:hover {
+        color: @sky
+      }
+    }
+
+    /* Docs */
+
+    --app-header: @crust;
+
+    --wh-color-primary-light-theme: @accent-color;
+    --wh-color-bg-main: @base;
+    --wh-color-text-main: @text;
+    --wh-color-text-secondary: @subtext0;
+    --wh-color-black-text-secondary: @subtext0;
+    --wh-color-black-text-pale: @subtext1;
+    --wh-color-substrate: @mantle;
+    --wh-color-backlight-main: @base;
+    --wh-color-backlight-secondary: @mantle;
+    --wh-color-white: @text;
+    --wh-color-black-text-main: @text;
+    --wh-color-border-default: @overlay1;
+    --wh-color-text-heading: @text;
+    --wh-color-breadcrumbs-active: @subtext1;
+    --wh-color-breadcrumbs-text: @overlay0;
+
+    --rs-color-danger: @red;
+    --wh-color-danger-t20: darken(@red, 40%);
+    --wh-color-danger-t40: darken(@red, 50%);
+    --rs-color-success: @green;
+    --wh-color-success-t20: darken(@green, 40%);
+    --wh-color-success-t40: darken(@green, 50%);
```

Comment:
These seem realllllyyyy dark. Use fade if you have to, or limit light/darkening to 5-15%.